config/default.yml

```diff
@@ -0,0 +1,5 @@
+collectors:
+  sources_per_collector:
```

Comment:
@agrare @bdunne how many sources per collector by default?

Reply:
I don't know how about CPU, but I can do some benchmark how much memory consumes one collecting of one source, and @Ladas have some benchmark I think, so it'll be something like 
`(max(single-source collector's RAM) - Rails app RAM) * number of sources) + Rails app RAM`

Reply:
I think the threading is still using GIL, correct? So I am pretty sure that can't go over 1 core.

As for memory, yeah the formula looks about right. (We'll need to make sure that all API calls are nicely paginated, so we don't keep much in memory and there are no spikes)